code/style.css

```diff
@@ -0,0 +1,183 @@
+body {
+  background: #fff;
+  color: white;
+  margin: 0;
+  font-family: 'Lato', Sans-Serif;
+}
+
+h1 {
+  margin: 0;
+}
+
+h2 {
+  font-size: 32px;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+}
+
+/****** HEADER ******/
+
+header {
+  position: relative;
+  color: #000;
+  display: flex;
+  font-family:'Arvo', Serif;
+  align-items: center;
+  justify-content: center;
+  height: 600px;
+}
+
+.header-title {
+  font-size: 72px;
+  text-align: center;
+}
+
+video {
+  display: block;
+  position: absolute;
+  z-index: -1;
+  margin: 0;
+  right: 0;
+  top: 0;
+  min-width: 100%;
+  height: 600px;
+  object-fit: cover;
+}
+
+/****** MAIN ******/
+
+main {
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin: 50px 0;
+}
+
+/* form */
+
+form {
+display: flex;
+flex-direction: column;
+background-color: #ff0025;
+margin: 20px 0;
+align-items: center;
+width: 100%;
+height: 680px;
+}
+
+.form-container {
+  width: 60%;
+ }
+
+.order {
+  margin: 15px;
+}
+
+.order input {
+  height: 50px;
+  width: 300px;
```

Comment:
You can achieve same result by setting two different properties, `font-size` and `padding` - that would be far more clean solution - `width` property can stay, but `height` is not necessary and should be avoided in this case